packages/paste-website/src/pages/customization/customizing-component-elements.mdx

```diff
@@ -33,7 +33,179 @@ export const getStaticProps = async () => {
 
 <content>
 
-We are currently working on this feature and providing support for it gradually across the entire suite of components. Watch this space!
+As a designer or developer of a Twilio Programmable App, you may need to customize a component to match your specific design requirements. Use Paste’s Customization Provider API to customize individual components or create entirely new ones based on existing components.
```

Comment:
Are you intentionally framing this page as only for folks creating twilio programmable apps? If I were a non-twilio customer user who happened upon our design system, I'd be turned away after reading the first sentence because I would think it's not built for me

Reply:
Kinda, and I'm OK with it. We positioned the whole section for Twilio Programmable apps https://paste.twilio.design/customization. It's still primarily a Twilio themed system for apps built by or on Twilio